website/api/trusted-setup-ceremony.md

```diff
@@ -0,0 +1,70 @@
+# Trusted Setup Security
+
+RISC Zero has run a trusted setup ceremony for our Groth16 prover/verifier. This ceremony secures our STARK Verify circuit so we can publish compact receipts for our general purpose zkVM to limited-memory environments like blockchains.
+
+You don't need to take our word for it that this is secure, though! This document will walk you through how to verify the security of our ceremony for yourself. This can also be used to aid in the detection of attacks related to our ceremony, e.g. it may help detect fraudlent actors publishing something they claim to be the RISC Zero STARK Verifier but which is actually malicious code.
+
+## Overview
+
+There are multiple steps to verifying the security of a trusted setup ceremony. Kobi Gurkan gave [a good list of what is required][kobi-bad-ceremony-list] in the form of a list of ways a malicious project could pretend to run a ceremony while secretly retaining a backdoor. Following this model, we will cover the following steps to verify our ceremony is secure:
+
+1. Verifying the circuit we are securing is what we intend to secure and does not include security holes.
+2. Verifying the transcript corresponds to this circuit
+3. Verifying contributors' attestations match the transcript
+4. Verifying the setup ceremony does not include security holes
+
+## The Circuit
+
+The circuit we are securing is the RISC Zero STARK Verify circuit, which is open source and [available on GitHub][stark-verify-circom] (along with [a short library][risc0-circom-library] it depends on).
+
+To ensure that the circuit itself does not have security holes, we have used a mixture of good software engineering practices, internal security reviews, and external audits. Stay tuned for detailed reports of our audit results!
+
+## The Transcript Matches the Circuit
+
+Our ceremony transcript is included in the `zkey` published on ceremony.pse.dev in the "Download ZKey" tab of the [RISC Zero STARK-to-SNARK Prover page][pse-risc0-ceremony]. (We mirror this file [here][TODO].) You can verify it matches the circuit using Circom and snarkjs:
```

Comment:
Is your thinking here to let this PR sit until we get this TODO resolved?

Reply:
Hmm, good call out, we _can't_ resolve this until the ceremony is fully finalized and published to the PSE website and then mirrored by us. That would put a fairly long hold on this.

I think instead we should drop "(We mirror this file here.)" for now and restore the line once it's actually mirrored; I'll make that change

Reply:
https://github.com/risc0/risc0/issues/1806